meadow/meadow/utils/new_book.py

```diff
@@ -0,0 +1,8 @@
+from meadow.models import Book
+from meadow.models import BookAuthor
+
+
+def create_new_book(link: str, descr: str, author_first: str, author_last: str):
+    a = BookAuthor(first_name=author_first, last_name=author_last)
+    a.save()
+    Book(download_link=link, description=descr, author=a)
```

Comment:
после создания объекта в памяти, его нужно записать в бд
```suggestion
    Book(download_link=link, description=descr, author=a).save()
```